package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "meteor-client-side",
+  "name": "meteor-client-side-official",
```

Comment:
I think it is a bit wried to call it official when it is not **Meteor** official package